spec/features/user_can_view_retired_items_but_cannot_order_item_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.